app/lib/shared/handler_helpers.dart

```diff
@@ -33,6 +33,7 @@ Future<void> runHandler(
   }
   handler = _redirectToHttpsWrapper(handler);
   handler = _logRequestWrapper(logger, handler);
+  handler = _cspHeaderWrapper(handler);
```

Comment:
Isn't this the same as self? Well, maybe not when we're on new domain :)

Reply:
The issue is with absolute URLs, and yeah, we may need to add an extra rule if we move to a new domain. But this PR already reduces the absolute URLs is dartdoc content, and hopefully the remaining ones are not heavily used.

Reply:
++